.travis.yml

```
@@ -12,6 +12,9 @@ matrix:
       env: BUILD_PHAR=true
     - php: 7.3
       env: WEBSITE=true
+    - php: 7.4
+    - php: 7.4
+      env: DEPENDENCIES=low
```

```suggestion
    - php: 8.0
```
We would except tests to also pass on 8.0

(not yet with DEPENDENCIES=low before we bump PDepend version to 2.9.0 which can run on PHP 8)

---

Yes, after I created this PR I found a `.travis.yml` file.

Next hours I will fix problems with dependencies, too.